versioned_docs/version-8.2/self-managed/zeebe-deployment/configuration/configuration.md

```diff
@@ -31,10 +31,10 @@ The default configuration is not suitable for a standalone gateway node. To run
 
 We provide templates that contain all possible configuration settings, along with explanations for each setting:
 
-- [`config/application.yaml` Standalone Broker (with embedded gateway)](https://github.com/camunda/zeebe/tree/stable/8.2/dist/src/main/config/application.yaml) - Default configuration containing only the most common configuration settings. Use this as the basis for a single broker deployment for test or development.
-- [`config/broker.standalone.yaml.template` Standalone Broker (with embedded gateway)](https://github.com/camunda/zeebe/tree/stable/8.2/dist/src/main/config/broker.standalone.yaml.template) - Complete configuration template for a standalone broker with embedded gateway. Use this as the basis for a single broker deployment for test or development.
-- [`config/broker.yaml.template` Broker Node (without embedded gateway)](https://github.com/camunda/zeebe/tree/stable/8.2/dist/src/main/config/broker.yaml.template) - Complete configuration template for a broker node without embedded gateway. Use this as the basis for deploying multiple broker nodes as part of a cluster.
-- [`config/gateway.yaml.template`](https://github.com/camunda/zeebe/tree/stable/8.2/dist/src/main/config/gateway.yaml.template) - Complete configuration template for a standalone gateway.
+- [`config/application.yaml` Standalone Broker (with embedded gateway)](https://github.com/camunda/zeebe/blob/8.2.0/dist/src/main/config/application.yaml) - Default configuration containing only the most common configuration settings. Use this as the basis for a single broker deployment for test or development.
```

Comment:
Is there no stable 8.2.x branch anymore that would always point to the latest patch for that minor version?

Reply:
If I understand it correctly, the stable branches may could in worst case contain an unreleased backported bugfix but they would be an easy target to link to, which doesn't need changing with every patch release, e.g. https://github.com/camunda/zeebe/blob/stable/8.2/dist/src/main/config/application.yaml

Reply:
For consistency with https://github.com/camunda/camunda-docs/pull/3743/files, we'll keep with a layout like `https://github.com/camunda/zeebe/tree/8.2.0/exporters/elasticsearch-exporter/src/main/resources`, with the exception of the `next` version, which will be updated for each release 👍 

Will add this set of links to my notes for release updates.